src/main/java/uk/co/ramp/event/AlertEventProcessor.java

```diff
@@ -89,6 +97,9 @@ int timeInStatus(AlertStatus newStatus) {
 
     MeanMax progressionData;
     switch (newStatus) {
+      case TESTED_POSITIVE:
+        progressionData = ImmutableMeanMax.builder().mean(1).max(1).build();
+        break;
```

Comment:
(At a glance,) this looks like it could be more concise:
```java
     case TESTED_POSITIVE:
     case TESTED_NEGATIVE:
     case ALERTED:
        progressionData = ImmutableMeanMax.builder().mean(1).max(1).build();
        break;
```

Reply:
Thanks @johnnonweiler good spot, will update in a separate PR. Thanks